src/bim360.ts

```diff
@@ -457,7 +458,7 @@ export class BIM360Client extends ForgeClient {
     async getItemDetails(projectId: string, itemId: string): Promise<IItemDetails> {
         let response = await this.get(`data/v1/projects/${encodeURIComponent(projectId)}/items/${encodeURIComponent(itemId)}`, {}, ReadTokenScopes);
         if (response.included && response.included.length > 0) {
-            return Object.assign(response.data.attributes, { id: response.data.id, type: response.data.type, derivative: response.included[0].relationships.derivatives.data.id });
+            return Object.assign(response.data.attributes, { id: response.data.id, type: response.data.type, derivative: response.included[0].relationships.derivatives.data.id, storage: response.included[0].relationships.storage.data.id });
```

Comment:
I noticed the commit message says "include storage ID if available". If we assume that the `response.included` array will have at least one item, will that item always include `.relationships.storage.data.id`? If so, this is fine, otherwise we should probably use something like the [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) to be on the safe side.